app/views/practices/form/overview.html.erb

```diff
@@ -16,6 +19,9 @@
       <%= render partial: "shared/messages", locals: {small_text: false} %>
       <section class="usa-section padding-top-0 p0-bottom introduction">
         <h1 class="font-sans-2xl line-height-46px margin-bottom-3">Overview</h1>
+        <div class="margin-top-2 margin-bottom-2">
+          <%= render partial: "practices/form/practice_last_updated", locals: {small_text: true} %>
+        </div>
```

Comment:
We can use the class margin-y-2 here, since both the top and bottom margin sizes are identical.

Reply:
Got it.

Reply:
let's just add this div inside the partial since I think we are using it every time we render the partial